pkg/virt-controller/watch/vm/vm.go

```diff
@@ -786,6 +787,57 @@ func (c *Controller) VMIAffinityPatch(vm *virtv1.VirtualMachine, vmi *virtv1.Vir
 	return err
 }
 
+func (c *Controller) VMITolerationsPatch(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) error {
```

Comment:
Should not start with capital letter.

Reply:
you're right, it's because I copied it from the affinity code